asv_bench/benchmarks/indexing.py

```diff
@@ -369,3 +369,47 @@ def time_assign_with_setitem(self):
             self.df[i] = np.random.randn(self.N)
 
 
+class Float64(object):
```

Comment:
I would personally keep those in the index benchmarks, as indexing an index is something completely different (implementation wise) than indexing a series/dataframe